.github/workflows/inductor-unittest.yml

```diff
@@ -0,0 +1,158 @@
+# Workflow: Inductor Unit Test
+# 1. runs unit tests for inductor.
+# 2. perfoms daily memory leak checks and reruns of disabled tests, scheduled at `29 8 * * *`.
+
+name: inductor-unittest
+on:
+  workflow_call:
+    inputs:
+      job_name:
+        description: 'A job name passed for concurrency group'
+        required: false
```

Comment:
Nit: As this is an optional parameter, it makes sense to provide a default value here too

Reply:
sounds good!